packages/x-data-grid-premium/src/tests/statePersistence.DataGridPremium.test.tsx

```diff
@@ -121,10 +117,7 @@ describe('<DataGridPremium /> - State persistence', () => {
         });
       });
 
-      let exportedState: GridInitialStatePremium = {};
-      await act(() => {
```

Comment:
Have you tried with an async function inside the act as well?

Reply:
Nope, based on testing, it doesn't seem that `act` makes sense here, as the call doesn't cause a re-render.